src/dsp_tools/xmllib/models/internal/values.py

```diff
@@ -282,7 +287,20 @@ def new(
     ) -> Richtext:
         check_and_warn_potentially_empty_string(value=value, res_id=resource_id, expected="string", prop_name=prop_name)
         converted_val = replace_newlines_with_tags(str(value), newline_replacement)
-        check_richtext_syntax(converted_val)
+
+        escaped_text = escape_reserved_xml_characters(converted_val)
+        num_ent = numeric_entities(escaped_text)
+        pseudo_xml = f"<ignore-this>{num_ent}</ignore-this>"
+        try:
+            _ = etree.fromstring(pseudo_xml)
+        except etree.XMLSyntaxError as err:
+            msg_str = (
+                f"The entered richtext value could not be converted to a valid XML.\n"
+                f"Original error message: {err.msg}\n"
+                f"Potential line/column numbers are relative to this text: {pseudo_xml}"
+            )
+            msg = MessageInfo(resource_id=resource_id, prop_name=prop_name, message=msg_str)
+            raise_input_error(msg)
```

Comment:
There is one thing I dislike: The code duplication in `value_checkers` and here. (To be fair, the duplication existed already before). I would prefer to have a separate function which does these steps:

```
escaped_text = escape_reserved_xml_characters(converted_val)
        num_ent = numeric_entities(escaped_text)
        pseudo_xml = f"<ignore-this>{num_ent}</ignore-this>"
        _ = etree.fromstring(pseudo_xml)
```

This new function can then be called from here, and from the  `value_checkers`.